addons/stock/report/report_stock_forecasted.py

```diff
@@ -130,9 +142,10 @@ def _prepare_report_line(self, quantity, move_out=None, move_in=None, replenishm
             'move_out': move_out,
             'move_in': move_in,
             'reservation': reservation,
+            'is_matched': is_matched,
```

Comment:
`move_to_match_id in [move_in_id, move_out_id]` directly here, to avoid 4 useless lines ?